File: src/main/java/io/github/mg138/tsbook/command/admin/Reload.java
package io.github.mg138.tsbook.command.admin;

import io.github.mg138.tsbook.Book;

import org.bukkit.command.CommandSender;

public class Reload {
    public static boolean call(CommandSender sender) {
        sender.sendMessage(Book.Companion.getCfg().translate.translate("messages.reload"));
        Book.inst.unload();
        Book.inst.load();
        sender.sendMessage(Book.Companion.getCfg().translate.translate("messages.reloaded"));
        return true;
    }
}

File: src/main/kotlin/io/github/mg138/tsbook/command/admin/Reload.kt
package io.github.mg138.tsbook.command.admin

import io.github.mg138.tsbook.Book
import io.github.mg138.tsbook.config.BookConfig
import org.bukkit.command.CommandSender

object Reload {
    fun call(sender: CommandSender): Boolean {
        sender.sendMessage(BookConfig.language.messages.reload)
        Book.inst.unload()
        Book.inst.load()
        sender.sendMessage(BookConfig.language.messages.reloaded)
        return true
    }
}